src/lib.rs

```diff
@@ -43,19 +46,6 @@ use indicatif::ProgressBar;
 pub struct Config {
     manifest_path: PathBuf,
     host: analysis::AnalysisHost,
-    assets: Vec<Asset>,
```

Comment:
Would it make sense to have something like

```
Config::new_from_manifest(
   &manifest_path,
   include!(concat!(env!("OUT_DIR"), "/asset.in"))
)
```

in `src/main.rs` instead? I'm not sure how we can make things *not* global due to the codegen.